### PR TITLE
Rails MarkIt 4: Topics crud

### PR DESCRIPTION
**Pull request by MelissaAJohnson**
screen shot 2016-08-14 at 8 24 49 am

Topics list on signed in user view.
screen shot 2016-08-13 at 8 18 55 pm

Topics show including edit and delete functions
screen shot 2016-08-13 at 8 19 45 pm

A better looking Sign Up form
screen shot 2016-08-13 at 8 20 32 pm

### CODE REVIEW

File: app/assets/javascripts/bookmarks.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/bookmarks.coffee]: Delete please

File: app/assets/javascripts/topics.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/topics.coffee]: Delete please

File: app/assets/stylesheets/bookmarks.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Bookmarks controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/bookmarks.scss]: Delete please

File: app/assets/stylesheets/topics.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Topics controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/topics.scss]: Delete please

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
```
Comment (wonderwoman13) [inline app/controllers/application_controller.rb]: Delete please

File: app/helpers/bookmarks_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module BookmarksHelper
```
Comment (brittanyjspringer) [inline app/helpers/bookmarks_helper.rb]: Delete please

File: app/helpers/topics_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module TopicsHelper
```
Comment (brittanyjspringer) [inline app/helpers/topics_helper.rb]: Delete please

File: app/models/bookmark.rb [hunk 8]
```diff
@@ -0,0 +1,3 @@
+class Bookmark < ApplicationRecord
```
Comment (brittanyjspringer) [inline app/models/bookmark.rb:1]: Delete please

File: spec/helpers/topics_helper_spec.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/helpers/topics_helper_spec.rb]: Delete please

File: spec/helpers/bookmarks_helper_spec.rb [hunk 10]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/helpers/bookmarks_helper_spec.rb]: Delete please

File: spec/views/bookmarks/edit.html.erb_spec.rb [hunk 11]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/views/bookmarks/edit.html.erb_spec.rb]: You can delete all View and Helper specs.

**Comment by brittanyjspringer** [comment]
Good morning, @melissajstudent. A couple of things: 
- Please keep the checkpoints separated. If there was a problem with Bookmarks, I'd hate to fail Topics. 
- Lots of empty files that need to be deleted. 
- Please indicate your test coverage from Simplecov. Current coverage: ##%
- Change the PR title to include the Checkpoint your are submitting. Example: Rails MarkIt 4 - Topics CRUD

I'll keep an eye out for a new commit. Thanks!

**Comment by MelissaAJohnson** [comment]
done - thanks for the SimpleCov reminder

**Comment by brittanyjspringer** [comment]
Thanks! Checking you off now.

**Pull request merged**